baselines/depthfl/README.md

```diff
@@ -0,0 +1,173 @@
+python -m depthfl.main --config-name="heterofl" exclusive_learning=true model_size=3 model.scale=false
+python -m depthfl.main --config-name="heterofl" exclusive_learning=true model_size=4 model.scale=false
+python -m depthfl.main fit_config.feddyn=false fit_config.kd=false  
+python -m depthfl.main fit_config.feddyn=false fit_config.kd=false  exclusive_learning=true model_size=1
+python -m depthfl.main fit_config.feddyn=false fit_config.kd=false  exclusive_learning=true model_size=2
+python -m depthfl.main fit_config.feddyn=false fit_config.kd=false  exclusive_learning=true model_size=3
+python -m depthfl.main fit_config.feddyn=false fit_config.kd=false  exclusive_learning=true model_size=4
+
+# table 3
+python -m depthfl.main fit_config.feddyn=false fit_config.kd=false fit_config.extended=false
+
+# table 4
+python -m depthfl.main fit_config.kd=false
+```
+
+The above commands would generate results in DepthFL paper. The numbers below are the results of a single run, and although they do not perfectly match the numbers recorded in the paper, they are very close.
+
+**Table 2** 
+
+100% (a), 75%(b), 50%(c), 25% (d) cases are exclusive learning scenario. 100% (a) exclusive learning means, the global model and every local model are equal to the smallest local model, and 100% clients participate in learning. Likewise, 25% (d) exclusive learning means, the global model and every local model are equal to the larget local model, and only 25% clients participate in learning.
+
+| Scaling Method | Dataset | Global Model | 100% (a) | 75% (b) | 50% (c) | 25% (d) | 
+| :---: | :---: | :---: | :---: | :---: | :---: | :---: | 
+| HeteroFL | CIFAR100 | 57.61 | 64.39 | 66.08 | 62.03 | 51.99 |
+| DepthFL (FedAvg) | CIFAR100 | 72.67 | 67.08 | 70.78 | 68.41 | 59.17 |
+| DepthFL | CIFAR100 | 76.06 | 69.68 | 73.21 | 70.29 | 60.32 |
+
+**Table 3** 
+
+Accuracy of global sub-models compared to exclusive learning on CIFAR-100.
+
+| Method | Algorithm | Classifier 1/4 | Classifier 2/4 | Classifier 3/4 | Classifier 4/4 |
+| :---: | :---: | :---: | :---: | :---: | :---: |
+| Width Scaling | Exclusive Learning | 64.39 | 66.08 | 62.03 | 51.99 |
+| Width Scaling | HeteroFL | 51.08 | 55.89 | 58.29 | 57.61 |
+
+| Method | Algorithm | Classifier 1/4 | Classifier 2/4 | Classifier 3/4 | Classifier 4/4 |
+| :---: | :---: | :---: | :---: | :---: | :---: |
+| Depth Scaling | Exclusive Learning | 67.08 | 68.00 | 66.19 | 56.78 |
+| Depth Scaling | InclusiveFL | 47.61 | 53.88 | 59.48 | 60.46 |
+| Depth Scaling | DepthFL (FedAvg)| 66.18 | 67.56 | 67.97 | 68.01 |
+
+**Table 4** 
+
+Accuracy of the global model with/without self distillation on CIFAR-100.
+
+| Distribution | Dataset | KD | Classifier 1/4 | Classifier 2/4 | Classifier 3/4 | Classifier 4/4 | Ensemble | 
+| :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
+| IID | CIFAR-100 | &cross; | 70.13 | 69.63 | 68.92 | 68.92 | 74.48 | 
+| IID | CIFAR-100 | &check; | 71.74 | 73.35 | 73.57 | 73.55 | 76.06 | 
```

Comment:
```suggestion
| Distribution | Dataset | KD | Classifier 1/4 | Classifier 2/4 | Classifier 3/4 | Classifier 4/4 | Ensemble | 
| :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
| IID | CIFAR-100 | &cross;<br>&check; | 70.13<br>71.74 | 69.63<br>73.35 | 68.92<br>73.57 | 68.92<br>73.55 | 74.48<br>76.06 | 

```

Reply:
I'm thinking your tables will look better if we use a "multi-row"-like presentation. I'm no expert with how to do this with markdown/html tables, but doing as above it seems to work. 

Could you do the same for the other tables?